packages/teams-js/src/internal/telemetry.ts

```diff
@@ -297,6 +297,7 @@ export const enum ApiName {
   Sharing_History_GetContent = 'sharing.history.getContent',
   Sharing_ShareWebContent = 'sharing.shareWebContent',
   StageView_Open = 'stageView.open',
+  StageView_Close = 'stageView.closeCurrentWindow',
```

Comment:
this should also match the capability and function names, so after the changes yuri asked for probably `'stageView.self.close'`